### PR TITLE
Fix spelling mistakes found by codespell and mwic

### PR DESCRIPTION
**Pull request by andersk**
## Changes

Fix spelling mistakes found by [codespell](https://github.com/codespell-project/codespell) and [mwic](https://jwilk.net/software/mwic).

## Testing

Reviewed `git show --word-diff`.

## Docs

No user-visible changes (beyond better spelling).

### CODE REVIEW

File: packages/astro/test/fixtures/astro-markdown-shiki/themes-custom/astro.config.mjs [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
...rective.vue","settings":{"foreground":"#77AAB3","fontStyle":"italic"}},{"scope":"meta.property-name.css","settings":{"foreground":"#7397DE"}},{"scope":"meta.property-value.css","settings":{"foreground":"#886CDB"}},{"scope":"meta.tag.other.html","settings":{"foreground":"#5F6488"}},{"scope":["punctuation"],"settings":{"foreground":"#5F6488"}},{"scope":["punctuation.accessor"],"settings":{"foreground":"#3788BE"}},{"scope":["punctuation.definition.string"],"settings":{"foreground":"#886CDB"}},{"scope":["punctuation.definition.tag"],"settings":{"foreground":"#8388AD"}},{"scope":["storage.type","storage.modifier"],"settings":{"foreground":"#3788BE"}},{"scope":["string"],"settings":{"foreground":"#886CDB"}},{"scope":["support"],"settings":{"foreground":"#7397DE"}},{"scope":["support.constant"],"settings":{"foreground":"#886CDB"}},{"scope":["support.function"],"settings":{"foreground":"#D26A5D","fontStyle":"italic"}},{"scope":["variable"],"settings":{"foreground":"#F19A8E","fontStyle":"italic"}},{"scope":["variable.other","variable.language","variable.function","variable.argument"],"settings":{"foreground":"#4E5377"}},{"scope":["variable.parameter"],"settings":{"foreground":"#77AAB3"}}]}`
```
Comment (delucis) [inline packages/astro/test/fixtures/astro-markdown-shiki/themes-custom/astro.config.mjs:2]: What’s the change here? Can’t find it in the diff.

File: packages/markdown/component/test/fixtures/astro-markdown-shiki/themes-custom/astro.config.mjs [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
...rective.vue","settings":{"foreground":"#77AAB3","fontStyle":"italic"}},{"scope":"meta.property-name.css","settings":{"foreground":"#7397DE"}},{"scope":"meta.property-value.css","settings":{"foreground":"#886CDB"}},{"scope":"meta.tag.other.html","settings":{"foreground":"#5F6488"}},{"scope":["punctuation"],"settings":{"foreground":"#5F6488"}},{"scope":["punctuation.accessor"],"settings":{"foreground":"#3788BE"}},{"scope":["punctuation.definition.string"],"settings":{"foreground":"#886CDB"}},{"scope":["punctuation.definition.tag"],"settings":{"foreground":"#8388AD"}},{"scope":["storage.type","storage.modifier"],"settings":{"foreground":"#3788BE"}},{"scope":["string"],"settings":{"foreground":"#886CDB"}},{"scope":["support"],"settings":{"foreground":"#7397DE"}},{"scope":["support.constant"],"settings":{"foreground":"#886CDB"}},{"scope":["support.function"],"settings":{"foreground":"#D26A5D","fontStyle":"italic"}},{"scope":["variable"],"settings":{"foreground":"#F19A8E","fontStyle":"italic"}},{"scope":["variable.other","variable.language","variable.function","variable.argument"],"settings":{"foreground":"#4E5377"}},{"scope":["variable.parameter"],"settings":{"foreground":"#77AAB3"}}]}`
```
Comment (delucis) [inline packages/markdown/component/test/fixtures/astro-markdown-shiki/themes-custom/astro.config.mjs:2]: Same question here.

File: packages/webapi/src/inheritance.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-export const inheritence = {
+export const inheritance = {
```
Comment (delucis) [inline packages/webapi/src/inheritance.ts:1]: Let’s avoid actual code changes. Doesn’t really matter if an internal API is misspelled and changing it with an automated tool is risky without any particular benefit.

File: packages/webapi/src/polyfill.ts [hunk 4]
```diff
@@ -83,7 +83,7 @@ import {
 } from './ponyfill'
 
 import { exclusions } from './exclusions'
-import { inheritence } from './inheritence'
```
Comment (delucis) [inline packages/webapi/src/polyfill.ts:86]: Changes in this file would also need to be reverted.

**Review by delucis — COMMENTED** [review]
Hi @andersk! The changes in code comments and Markdown files look OK but I’m less convinced by the automated changes to code. Left some comments.

File: packages/astro/test/fixtures/astro-markdown-shiki/themes-custom/astro.config.mjs (hunk 1 above)
Comment (andersk) [reply packages/astro/test/fixtures/astro-markdown-shiki/themes-custom/astro.config.mjs:2]: It’s `"constant.charcter.escape"` → `"constant.character.escape"`, as in `packages/astro/test/fixtures/astro-component-code/src/assets/serendipity-morning.json`.

File: packages/webapi/src/inheritance.ts (hunk 3 above)
Comment (andersk) [reply packages/webapi/src/inheritance.ts]: I’ve split this out into #5634. It’s easy to review with `git grep inheritence` and `git ls-files | grep inheritence`.

**Review by delucis — APPROVED** [review]
Thanks @andersk!

**Pull request merged**